templates/pkg/resource/sdk_find_read_many.go.tpl

```diff
@@ -51,6 +58,19 @@ func (rm *resourceManager) sdkFind(
 	return &resource{ko}, nil
 }
 
+// requiredFieldsMissingFromReadManyInput returns true if there are any fields
+// for the ReadMany Input shape that are required but not present in the
+// resource's Spec or Status
+func (rm *resourceManager) requiredFieldsMissingFromReadManyInput(
+	r *resource,
+) bool {
+{{- if $customCheckMethod := .CRD.GetCustomCheckRequiredFieldsMissingMethod .CRD.Ops.ReadMany }}
+    return rm.{{ $customCheckMethod }}(r)
```

Comment:
@jaypipes hope this is properly formatted. I tried your suggestion from last PR, but that adjusted the spacing between the comments within the template as well

Reply:
It's OK :) We run `go fmt` against the outputted code anyway, so even though no, it's still using spaces, it will get corrected ex post facto...